checks/containers/sarus/check_osu_benchmarks.py

```diff
@@ -1,8 +1,20 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import pathlib
+import sys
+
 import reframe as rfm
 import reframe.utility.sanity as sn
 
+sys.path.append(str(pathlib.Path(__file__).parent.parent.parent / 'mixins'))
+
+from sarus_extra_launcher_options import SarusExtraLauncherOptionsMixin
+
 
-class BaseCheck(rfm.RunOnlyRegressionTest):
+class BaseCheck(SarusExtraLauncherOptionsMixin, rfm.RunOnlyRegressionTest):
```

Comment:
you may want to move to a more recent version but for the sake of time, let's skip updating the version.

Reply:
This is a matter of the image that we are using